test/integration/integration_admin_test.cc

```diff
@@ -266,6 +266,56 @@ TEST_P(IntegrationAdminTest, Admin) {
   }
 }
 
+// TEST_P(IntegrationAdminTest, AdminSetStreaming) {
+//  initialize();
+//
+//  auto callback = [this](absl::string_view, Http::HeaderMap&, Buffer::Instance&,
+//                         Server::AdminStream& admin_stream) -> Http::Code {
+//
+//    admin_stream.setEndStreamOnComplete(false);
+//    return Http::Code::OK;
+//  };
+//
+//  // Test removable handler.
+//  EXPECT_TRUE(
+//      test_server_->server().admin().addHandler("/foo/bar", "hello", callback, true, false));
+//
+//  BufferingStreamDecoderPtr response = IntegrationUtil::makeSingleRequest(
+//      lookupPort("admin"), "GET", "/foo/bar", "", downstreamProtocol(), version_);
+//
+//  EXPECT_TRUE(response->complete());
+//  EXPECT_STREQ("200", response->headers().Status()->value().c_str());
+//  EXPECT_EQ(spdlog::level::trace, Logger::Registry::getLog(Logger::Id::assert).level());
+//  // TODO (@trabetti) : what can be checked on response headers?
+//}
```

Comment:
What's the story here?

Reply:
tried to see if there's a way to test end_stream = false. but then the response never returns.